segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/chunklayer/UtilsWrapperTests.java

```diff
@@ -275,4 +284,191 @@ public void testInvalidParameters() throws Exception {
                 () -> wrapper.getExtendedChunkInfoList(null, false),
                 ex -> ex instanceof NullPointerException);
     }
+
+    /**
+     * Sanity test to check if a new chunk was created with the provided content.
+     * @throws Exception Throws exception in case of any error.
+     */
+    @Test
+    public void testSanityCreateChunk() throws Exception {
+        @Cleanup
+        BaseChunkStorage spyChunkStorage = spy(new InMemoryChunkStorage(executorService()));
+        Exception exceptionToThrow = new ChunkStorageException("test", "Test Exception", new IOException("Test Exception"));
+        doThrow(exceptionToThrow).when(spyChunkStorage).doCreateWithContent(any(), anyInt(), any());
+
+        testSanity(spyChunkStorage, "Test Exception", ChunkStorageException.class);
+    }
+
+    /**
+     * Sanity test to verify if the chunk info was correctly retrieved for given chunk.
+     * @throws Exception Throws exception in case of any error.
+     */
+    @Test
+    public void testSanityGetChunkInfo() throws Exception {
+        @Cleanup
+        BaseChunkStorage spyChunkStorage = spy(new InMemoryChunkStorage(executorService()));
+        Exception exceptionToThrow = new ChunkStorageException("test", "Test Exception", new IOException("Test Exception"));
+        val clazz = ChunkStorageException.class;
+        doThrow(exceptionToThrow).when(spyChunkStorage).doGetInfo(any());
+
+        testSanity(spyChunkStorage, "Test Exception", clazz);
+    }
+
+    /**
+     * Sanity test to open chunk to write/ make modifications.
+     * @throws Exception Throws exception in case of any error.
+     */
+    @Test
+    public void testSanityWriteIntoChunk() throws Exception {
+        @Cleanup
+        BaseChunkStorage spyChunkStorage = spy(new InMemoryChunkStorage(executorService()));
+        Exception exceptionToThrow = new ChunkStorageException("test", "Test Exception", new IOException("Test Exception"));
+        val clazz = ChunkStorageException.class;
+        doThrow(exceptionToThrow).when(spyChunkStorage).doOpenWrite(any());
+
+        testSanity(spyChunkStorage, "Test Exception", clazz);
+
+    }
+
+    /**
+     * Sanity test to check if the chunk exists.
+     * @throws Exception Throws exception in case of any error.
+     */
+    @Test
+    public void testSanityCheckChunkExists() throws Exception {
+        @Cleanup
+        BaseChunkStorage spyChunkStorage = spy(new InMemoryChunkStorage(executorService()));
+        val clazz = IllegalStateException.class;
+
+        doReturn(false).when(spyChunkStorage).checkExists(any());
+
+        testSanity(spyChunkStorage, "The given chunk doesn't exist!", clazz);
+    }
+
+    /**
+     * Test to verify if the bytes read are equal to the dataSize of the test data.
+     * @throws Exception Throws exception in case of any error.
+     */
+    @Test
+    public void testSanityDataEquals() throws Exception {
```

Comment:
I think it is better to use more descriptive names like
`testSanityDataEquals`-> `testSanityDataEqualityFails`
that suggest that it is negative test.